### PR TITLE
Tweaks README to fix typo and be more visible

### PR DESCRIPTION
**Pull request by ellis-bigelow**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/401)

### CODE REVIEW

**Comment by ellis-bigelow** [comment]
I think 0.9 is busted currently.

File: README.md [hunk 1]
```diff
@@ -13,10 +15,10 @@ KFServing encapsulates the complexity of autoscaling, networking, health checkin
 * [KFServing 101 Tech Talk](https://www.youtube.com/watch?v=hGIvlFADMhU).
 * This project is an evolution of the [original proposal in the Kubeflow repo](https://github.com/kubeflow/kubeflow/issues/2306). 
 
-### Prerequisits
+### Prerequisites
 KNative Serving and Istio should be available on Kubernetes Cluster.
-- Istio Version: v1.1.7 + 
-- Knative Version: v0.8.0 +
+- Istio Version: v1.1.7+ 
+- Knative Version: v0.8.x
```
[the remark this answers is not in the document]
Comment (animeshsingh) [reply README.md]: Yes - should be v0.9

File: README.md [hunk 2]
```diff
@@ -35,5 +37,3 @@ pip install kfserving
 
 ### Contribute
 * [Developer Guide](/docs/DEVELOPER_GUIDE.md).
-
-![KFServing](./docs/diagrams/kfserving.png)
```
Comment (animeshsingh) [inline README.md]: Good you removed the diag - need to do a quick followup on this

**Comment by ellis-bigelow** [comment]
I just moved it upwards to be more prominent.

**Pull request merged**